sql-delta-import/src/main/scala/com/razorpay/spark/jdbc/JDBCImport.scala

```diff
@@ -150,25 +150,31 @@ class JDBCImport(
   private def readJDBCSourceInParallel(): DataFrame = {
 
     if (importConfig.splitBy.nonEmpty) {
+      val defaultString="defaultValue"
```

Comment:
is this used somewhere? I think a better option would be to set it as empty string. `""`

Reply:
```suggestion
      val defaultString = ""
```

Reply:
Its not used anywhere, will set it to empty